src/hooks/use-calendar-actions.test.ts

```diff
@@ -202,16 +202,18 @@ describe('use calendar actions', () => {
 			initialProps: [calendarItem]
 		});
 
-		expect(result.current.length).toBe(5);
+		expect(result.current.length).toBe(7);
 		expect(result.current).toStrictEqual([
 			expect.objectContaining({ id: FOLDER_ACTIONS.NEW }),
 			expect.objectContaining({ id: FOLDER_ACTIONS.EDIT }),
 			expect.objectContaining({ id: FOLDER_ACTIONS.SHARE }),
 			expect.objectContaining({ id: FOLDER_ACTIONS.SHARE_URL }),
-			expect.objectContaining({ id: FOLDER_ACTIONS.FIND_SHARES })
+			expect.objectContaining({ id: FOLDER_ACTIONS.FIND_SHARES }),
+			expect.objectContaining({ id: FOLDER_ACTIONS.EXPORT_ICS }),
+			expect.objectContaining({ id: FOLDER_ACTIONS.UPLOAD })
 		]);
 	});
-	test('shared account calendar has 4 actions', () => {
+	test('shared account calendar has 5 actions', () => {
```

Comment:
the test description should report "6 actions" instead of 5

Reply:
updated